src/js/components/week.jsx

```diff
@@ -21,22 +21,25 @@ export default class Week extends React.Component {
       timeslotProps,
       selectedTimeslots,
       disabledTimeslots,
+      renderDays,
     } = this.props;
 
     return weekToRender.map((day, index) => {
       let formattedDate = helpers.getMomentFromCalendarJSDateElement(day);
-      return (
-        <Day
-          key = { index }
-          onTimeslotClick = { this._onTimeslotClick.bind(this) }
-          initialDate = { initialDate }
-          timeslots = { timeslots }
-          timeslotProps = { timeslotProps }
-          selectedTimeslots = { selectedTimeslots }
-          disabledTimeslots = { disabledTimeslots }
-          momentTime = { formattedDate }
-          />
-      );
+      if (renderDays[formattedDate.format('dddd').toLowerCase()]){
```

Comment:
get `formattedDate.format('dddd').toLowerCase()` into a variable for better readability